.github/workflows/native-build.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           matrix = {
             "run": [
               {
-                "runner": "macos-11",
+                "runner": "macos-12",
                 "tag": "macosx_10_9_universal2"
```

Comment:
Doesn't this tag need to change too?  New image means new build means new osx minimum version....